### PR TITLE
Add Additional Checks on Assets Folder

### PR DESCRIPTION
**Pull request by nlordell**
This PR adds additional tests on the JSON files in the `assets/` directory to ensure that:
1. Only canonical addresses are used
2. That the `networkAddresses` object is ordered
3. Networks are included for every contract

This led to some changes :point_down:.

### CODE REVIEW

File: src/assets/v1.3.0/create_call.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "255": "0xB19D6FFc2182150F8Eb585b79D4ABcd7C5640A9d",
     "280": "0xcB8e5E438c5c2b45FbE17B02Ca9aF91509a8ad56",
     "288": "0xB19D6FFc2182150F8Eb585b79D4ABcd7C5640A9d",
+    "291": "0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4",
```
Comment (nlordell) [inline src/assets/v1.3.0/create_call.json:53]: This was missing. Verified here: https://explorer.orderly.network/address/0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4?tab=contract

File: src/assets/v1.3.0/sign_message_lib.json [hunk 2]
```diff
@@ -153,8 +152,10 @@
     "73799": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "80001": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "84531": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+    "84532": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
     "200101": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "200202": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "333999": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```
Comment (nlordell) [inline src/assets/v1.3.0/sign_message_lib.json:158]: Missing, but unable to verify. Using what the address should be based on other contracts.

File: src/assets/v1.3.0/simulate_tx_accessor.json [hunk 3]
```diff
@@ -58,6 +58,7 @@
     "336": "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
     "338": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
     "420": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
+    "424": "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
```
Comment (nlordell) [inline src/assets/v1.3.0/simulate_tx_accessor.json:61]: Missing address, verified https://explorer.publicgoods.network/address/0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da?tab=contract

File: src/assets/v1.3.0/simulate_tx_accessor.json [hunk 4]
```diff
@@ -144,6 +145,7 @@
     "54211": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
     "56288": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
     "57000": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
+    "58008": "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
```
Comment (nlordell) [inline src/assets/v1.3.0/simulate_tx_accessor.json:148]: Contract is not verified, but was deployed: https://explorer.sepolia.publicgoods.network/address/0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da?tab=contract

**Review by ElvisKrop — COMMENTED** [review]
Awesome :rocket: 
I like it! :grin:

**Review by mmv08 — APPROVED** [review]
This was missing for so long!

**Review by M98765alizad — COMMENTED** [review]
لطفا خواهشن نسخه 1.3.0 safe global را ادرس امن ولت را بتونم تایید کنم تا از دارایهام استفاده کنم ممنون

**Pull request merged**